### DISCUSSION TITLE
jalali (Persian) date picker and calendar

### DISCUSSION THREAD

**Discussion by aliakbarebrahimy** [tusen-ai/naive-ui#4695] [Ideas]
Hello, is it possible to set the Jalali calendar for the date picker component and similar items?

**Comment by jahnli** [comment]
Need 'Persian' language?

**Reply by aliakbarebrahimy** [reply]
hi jahni , Support for Farsi language and right-to-left arrangement is one of our most important needs, what we want is a solar calendar and date/time picker
https://en.wikipedia.org/wiki/Solar_Hijri_calendar
thanks

**Comment by jahnli** [comment]
@aliakbarebrahimy  I'm not sure if the current component has a Persian translation, if not, I'll try a Persian translation

**Comment by jahnli** [comment]
@aliakbarebrahimy   https://github.com/tusen-ai/naive-ui/pull/4714
